security/oc_pstat_internal.h

```diff
@@ -0,0 +1,54 @@
+/****************************************************************************
```

Comment:
merge `oc_pstat.h` and `oc_pstat_internal.h`, all headers in the security folder are internal, I've just haven't renamed them all yet.